addons/default/visiosoft/advs-module/src/Http/Controller/advsController.php

```diff
@@ -784,16 +785,6 @@ public function store
             $adv->is_get_adv = $request->is_get_adv;
             $adv->save();
 
-            //algolia Search Module
```

Comment:
Why algolia have been removed ?

Reply:
it's 2 years old we need to make it with events